section2/exercises/ex3.rb

```diff
@@ -0,0 +1,41 @@
+puts "You enter a dark room with two doors.  Do you go through door #1 or door #2?"
```

Comment:
Nice work, I'd especially like to highlight your Ruby syntax and style - referring to the conventions around indentation, spaces, whitespace, etc. - since you are consistently following those conventions, your code is very readable.